keras_nlp/src/layers/modeling/reversible_embedding.py

```diff
@@ -109,12 +109,13 @@ def build(self, inputs_shape=None):
         super().build(inputs_shape)
 
         if not self.tie_weights:
```

Comment:
just combine into `if not self.tie_weights and self.quantization_mode != "int8":`?

Reply:
Fixed